pkgs/by-name/vl/vlc-bittorrent/package.nix

```diff
@@ -0,0 +1,66 @@
+{ autoconf
+, autoconf-archive
+, automake
+, boost
+, fetchFromGitHub
+, gcc
+, gnumake
+, lib
+, libtool
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
```

Comment:
`lib` should be the first input in package expressions (as a matter of nixpkgs' style).